index.js

```diff
@@ -568,6 +568,34 @@ class Squeakquel extends Datastore {
             .then(items => Promise.all(items.map(item =>
                 decodeFromDialect(this.client.getDialect(), item, model))));
     }
+
+    /**
+     * Run raw query on the datastore
+     * @param  {Object}        config                Configuration object
+     * @param  {Array<Object>} [config.queries]      Map of database type to query
+     * @param  {String}        [config.table]        Table name
+     * @param  {Object}        [config.replacements] Parameters to replace in the query
+     * @param  {Boolean}       [config.rawResponse]  Return raw response without binding to model
+     */
+    _query(config) {
+        const table = this.tables[config.table];
+        const model = this.models[config.table];
+        const query = config.queries.find(q => q.dbType === this.client.getDialect()).query;
```

Comment:
Probably will need some null checks here

Reply:
Added, thank you